MetaMask/metamask-extension
# feat: add logo for BounceBit network

## **Description**



Adds an icon for the BounceBit network



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25082?quickstart=1)

## **Related issues**


## **Manual testing steps**

1. Go to wallet or chainid or chainlist
2. Add the BounceBit network, chain id 6001
3. Verify the network has an icon in metamask

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


I have read the CLA Document and I hereby sign the CLA


## **Pre-merge reviewer checklist**
